src/Core/WinSWCore/ServiceDescriptor.cs

```diff
@@ -126,16 +126,8 @@ private int SingleIntElement(XmlNode parent, string tagName, int defaultValue)
 
         private TimeSpan SingleTimeSpanElement(XmlNode parent, string tagName, TimeSpan defaultValue)
         {
-            var e = parent.SelectSingleNode(tagName);

```

Comment:
There is similar pattern for Int above, but it is not being used to parse config entries